### ISSUE TITLE
[change] Eliminate redundancy of header HTML

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/openwisp-wifi-login-pages#314]
The header HTML is kind of duplicated, once is a desktop version and one is a mobile version.

I wonder if we can avoid this and handle everything with CSS.

It's kind of painful to customize this part because of this duplication (double amount of work).

**Comment by Red-0111** [comment]
Hi, I would like to work on this issue, please tell me how to proceed :)

**Comment by codesankalp** [comment]
Hi @Red-0111, Please check `client/components/header/`.
In this headers are defined for mobile and desktop separately.
**mobile**:
https://github.com/openwisp/openwisp-wifi-login-pages/blob/f2af7f1940aa6f41d2b2d2b35567c016382396b2/client/components/header/header.js#L145-L148
**desktop**:
https://github.com/openwisp/openwisp-wifi-login-pages/blob/f2af7f1940aa6f41d2b2d2b35567c016382396b2/client/components/header/header.js#L59-L62

In this issue, it is mentioned to get rid of one header using CSS.

**Comment by Red-0111** [comment]
Yes please give me some time I am trying to figure it out.

**Comment by codesankalp** [comment]
> Yes please give me some time I am trying to figure it out.

Great, If you need any help feel free to ask :+1:

**Comment by iamvaishnav** [comment]
Is anyone working on this?

**Comment by Red-0111** [comment]
> Is anyone working on this?

Yes, it would be great to have some help :)

**Comment by codesankalp** [comment]
> > Is anyone working on this?
> 
> Yes, it would be great to have some help :)

What issues are you facing @Red-0111?

**Comment by Red-0111** [comment]
> > > Is anyone working on this?
> > 
> > Yes, it would be great to have some help :)
> 
> What issues are you facing @Red-0111?

While running the headless test for yarn the test suites fail and it shows command failed with exit code 1

**Comment by codesankalp** [comment]
Have you tried the correct way of running browser-test as described in README?

@Red-0111 If you have implemented this, please make a draft pull request. I will review the PR and then can tell more about the error.

**Comment by Red-0111** [comment]
> Have you tried the correct way of running browser-test as described in README?
> 
> @Red-0111 If you have implemented this, please make a draft pull request. I will review the PR and then can tell more about the error.
> 

Yes I followed the same steps as in the README. I was installing dependencies so what should I make the pull request for?

**Comment by codesankalp** [comment]
> I was installing dependencies so what should I make the pull request for?

I thought you have implemented the changes and now trying to run tests before creating PR. It will be better if you can provide some details about the error (screenshots). Also before running tests have you run the command `yarn setup`?

**Comment by nemesisdesign** [comment]
@Red-0111 I suggest using the OpenWISP support chat to coordinate better.

**Comment by Red-0111** [comment]
> @Red-0111 I suggest using the [OpenWISP support chat](https://gitter.im/openwisp/general) to coordinate better.

Yeah sure

**Comment by mrgear111** [comment]
@pandafy will work on this.. ok?

**Comment by suhaib-ilahi** [comment]
Hey @prathmeshkulkarni-coder, seems you have changed the configuration files.

**Comment by suhaib-ilahi** [comment]
Hey @nemesifier, is this issue can be assigned to me? Can I work on this?

**Comment by suhaib-ilahi** [comment]
Maybe, we should address this in a separate PR.
